app/models/bot/tagger.rb

```diff
@@ -0,0 +1,72 @@
+require 'json'
+
+class Bot::Tagger < BotUser
+  check_settings
+  class Error < ::StandardError
+  end
+
+  def self.get_tag_text(tag_id,auto_tag_prefix,ignore_autotags)
+    tag=TagText.find(tag_id).text
```

Comment:
This will crash (and we'll get a Sentry alert) if there is no `TagText` with this ID. Is this the intended behavior? We can simply ignore it:

```
tag = TagText.find_by_id(tag_id)
return nil if tag.nil?
tag = tag.text
```

Reply:
`Model.find(id)` -> raises an exception if object doesn't exist
`Model.find_by_id(id)` -> returns nil if object doesn't exist

Reply:
Incorporated into the next if

```
    tag=TagText.find_by_id(tag_id).text
    if tag.nil? || (ignore_autotags && tag[0]==auto_tag_prefix)
      tag=nil
```